internal/sdk/cloudian/sdk_test.go

```diff
@@ -1,16 +1,57 @@
 package cloudian
 
 import (
+	"context"
 	"encoding/json"
 	"errors"
 	"fmt"
 	"math/rand"
+	"net/http"
+	"net/http/httptest"
 	"reflect"
 	"strings"
 	"testing"
 	"testing/quick"
+
+	"github.com/google/go-cmp/cmp"
 )
 
+func TestGetGroup(t *testing.T) {
+	expected := Group{
+		GroupID: "QA",
+	}
+
+	mockServer := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+		w.Write([]byte(`{"groupId": "QA"}`))
+	}))
+	defer mockServer.Close()
+
+	mockClient := &http.Client{}
+
+	_, err := http.NewRequestWithContext(context.TODO(), http.MethodGet, mockServer.URL, nil)
+	if err != nil {
+		t.Fatalf("Failed to create request: %v", err)
+	}
+
+	cloudianClient := Client{
+		baseURL:    mockServer.URL,
+		httpClient: mockClient,
```

Comment:
As it's not used elsewhere?

```suggestion
	_, err := http.NewRequestWithContext(context.TODO(), http.MethodGet, mockServer.URL, nil)
	if err != nil {
		t.Fatalf("Failed to create request: %v", err)
	}

	cloudianClient := Client{
		baseURL:    mockServer.URL,
		httpClient: &http.Client{},
```